core/src/types.ts

```diff
@@ -320,13 +320,14 @@ export declare interface VaultTransactionBase extends Vault, VaultTransactionFee
 
 export declare interface VaultTransactionLiquidated extends VaultBase {
     state: 'liquidated';
-    liqudiationDate: Date;
-    transactionHash: string;
-    auctionId: string;
+    liqudiationDate?: Date;
+    transactionHash?: string;
+    auctionId?: string;
```

Comment:
Why was this changed? If the vault is liquidated it should have all those values, they are not optional